base_view_inheritance_extension/models/res_users.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 Therp BV <https://therp.nl>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from openerp import api, models
+
+
+class ResUsers(models.Model):
+    _inherit = 'res.users'
+
+    @api.multi
+    def unlink(self):
+        """
+        Finds any views assigned specifically to this user and deletes them.
+        """
+        self.env['ir.ui.view'].sudo().with_context(
+            active_test=False).search([
+                ('user_ids', 'in', self.ids)]).with_context(
+                    active_test=True).filtered(
```

Comment:
what do you expect the second `with_context` to do?

Reply:
here I expected a reply in line with https://github.com/hbrunn/server-tools/pull/7#discussion_r313504264, not to remove it